dev/buildtool/bom_base.yml

```diff
@@ -1,3 +1,8 @@
+artifactSources:
+  debianRepository: https://us-apt.pkg.dev/projects/spinnaker-community
```

Comment:
We currently reference this file during `buildtool.sh build_bom ...` per [README.md](https://github.com/spinnaker/buildtool#build-bom).

It would be nicer if this file was deleted and default configuration maintained in python using constants.

Reply:
Indeed we don't need it - https://github.com/spinnaker/buildtool/pull/186/commits/67f2235e8c5cd70b35cf0d165cdc06ce4dde8dfd#diff-fb3a8caa8e42b779c50f869dfe0d462b3052d714bc571654bf2aa30433f4c19bL1-L5